RELEASE_NOTES.md

```diff
@@ -31,6 +31,11 @@ v1.0.0-alpha.X
   provide more context when using it as a starting point for an
   OpenAssetIO integration.
 
+- Made `BatchElementError` a copyable type.
+
+- `BatchElementErrorCallback` moved from the top level `openassetio`
```

Comment:
[fix] I think this should be in the "Breaking Changes" section.  Also good to link the issue.

Reply:
Ah yep, my bad. 
https://github.com/OpenAssetIO/OpenAssetIO/pull/864/commits/aa4dccea40f04c53a322f5cf3e6ca7f33115607c